dspace/config/modules/oai.cfg

```diff
@@ -42,7 +42,20 @@ oai.bitstream.baseUrl = ${dspace.ui.url}
 oai.config.dir = ${dspace.dir}/config/crosswalks/oai
 
 # Description
-oai.description.file = ${dspace.dir}/config/crosswalks/oai/description.xml
+oai.description.file.0 = ${dspace.dir}/config/crosswalks/oai/description.xml
```

Comment:
use LINDAT's defaults

Reply:
https://github.com/ufal/clarin-dspace/blob/clarin/dspace/config/crosswalks/oai/description-olac.xml ?

Reply:
https://github.com/ufal/clarin-dspace/blob/8a6ba5c98547942d7115b74cf4978e0b29ca50e4/utilities/project_helpers/config/local.conf.dist#L299

Reply:
except replace myself

Reply:
oh of course.. you commented on wrong line. Will replace.

Reply:
@vidiecan  replaced, but help.mail will have clash and will be replaced. Milan said we are not using lr.* config values, I am not sure what is the best course then.